### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by acoyfellow** [benweier/connect-session-firebase#126]
I typically use this library. Running into issues right now. 
running `npm install firebase-admin connect-session-firebase --save` yields:

```
npm ERR! code ERESOLVE
npm ERR! ERESOLVE unable to resolve dependency tree
npm ERR! 
npm ERR! While resolving: project@0.0.1
npm ERR! Found: firebase-admin@9.5.0
npm ERR! node_modules/firebase-admin
npm ERR!   firebase-admin@"^9.5.0" from the root project
npm ERR! 
npm ERR! Could not resolve dependency:
npm ERR! peer firebase-admin@"~5.13.0" from connect-session-firebase@5.13.0
npm ERR! node_modules/connect-session-firebase
npm ERR!   connect-session-firebase@"^5.13.0" from the root project
npm ERR! 
npm ERR! Fix the upstream dependency conflict, or retry
npm ERR! this command with --force, or --legacy-peer-deps
npm ERR! to accept an incorrect (and potentially broken) dependency resolution.
npm ERR! 
npm ERR! See /root/.npm/eresolve-report.txt for a full report.

npm ERR! A complete log of this run can be found in:
npm ERR!     /root/.npm/_logs/2021-03-21T14_24_06_501Z-debug.log
```

**Comment by acoyfellow** [comment]
as suggested, `npm i --legacy-peer-deps` resolves, but how can we solve the reccomended install option?

**Comment by fnisarm** [comment]
> as suggested, `npm i --legacy-peer-deps` resolves, but how can we solve the reccomended install option?

This doesn't fix the issue in my case https://github.com/firebase/firebase-admin-node/issues/1915

**Comment by benweier** [comment]
@acoyfellow @fnisarm apologies for not noticing this issue earlier. I have cleaned up the package dependencies plus other tooling, and published `11.0.0` to npm. This version now expects any `firebase-admin@11.x.x` peer
